mock/etc/mock/site-defaults.cfg

```diff
@@ -378,6 +378,10 @@ config_opts['bootstrap_module_install'] = []
 #
 # Proxy settings (https_proxy, ftp_proxy, and no_proxy can also be set)
 # config_opts['http_proxy'] = 'http://localhost:3128'
+config_opts['http_proxy']  = os.getenv("http_proxy")
+config_opts['https_proxy'] = os.getenv("https_proxy")
+config_opts['no_proxy']    = os.getenv("no_proxy")
```

Comment:
in site-defaults.cfg everything should be actually commented out. No real code there. Just inform the user about default. So these lines are fine, just put # in front of them. The real code should go to mock/py/mockbuild/util.py to `get_proxy_environment()`

Reply:
In `get_proxy_environment`, not in `setup_default_config_opts` ???